.travis.yml

```diff
@@ -63,6 +67,7 @@ before_script:
 script:
   - composer validate
   - bin/phpunit -c ./phpunit.xml.dist --coverage-clover=./build/coverage/coverage.clover
+  - if [[ $TEST_PHPSTAN ]]; then vendor/bin/phpstan analyse -l1 src; fi;
```

Comment:
This won't fix php version requirements as phpstan breaks dependencies installation